nuxt.config.js

```diff
@@ -155,6 +155,8 @@ export default {
           warning: colors.amber.base,
           error: colors.deepOrange.accent4,
           success: colors.green.accent3,
+          toolbar: colors.indigo.lighten1,
```

Comment:
Oh I didn't know you could add more color vars, that's awesome

Reply:
Yup! It makes it much easier to change the overall theme.